install_sct.bat

```diff
@@ -0,0 +1,66 @@
+:: Usage: install_sct.bat <version>

```

Comment:
Oh! Thank you. That whole SO question + answer is useful, too, as it lead me to [this batch guide](https://www.tacktech.com/display.cfm?ttid=307). 😄

Reply:
Follow-up to this: PyCharm [automatically adds `@REM`](https://stackoverflow.com/questions/64730481/how-to-change-toggle-comment-from-rem-to-rem-in-vs-code) when I use the commenting keyboard shortcut. I was curious about [`@REM` vs `REM`](https://softwareengineering.stackexchange.com/a/343864):

```batch
REM This will be shown during execution
@REM This will not!
@ECHO OFF
REM Nor this!
```

Probably Batch 101, but TIL. :)

Reply:
Oops. TIL to be careful about using `@` in git commits, lest I accidentally tag https://github.com/Rem. 😆